### PR TITLE
Add to flow run logs command option to print logs order by desc 

### LINKED ISSUE #8000: Add --reverse option to the prefect flow-run logs CLI
Follow-up to https://github.com/PrefectHQ/prefect/pull/7982

### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar request and didn't find it.
- [X] I searched the Prefect documentation for this feature.

### Prefect Version

2.x

### Describe the current behavior

The CLI command: `prefect flow-run logs <flow_run_id>` shows all logs in chronological order.

### Describe the proposed behavior

It would be great to add a `--reverse` option to show logs in reverse chronological order (print most recent log first). 

```bash
prefect flow-run logs <flow_run_id> [--reverse]
```

### Example Use

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lounis89**
This PR is a try to resolve this issue [#8000](https://github.com/PrefectHQ/prefect/issues/8000)
I tested my code with cli locally with :
```
prefect flow-run logs <flow_run_id> [--reverse] 
```

### Example

```
prefect flow-run logs 41d847f6-a489-4a9e-b7ba-c27840d81776           
2023-02-23 08:46:32.044 | INFO    | Flow run 'winged-roadrunner' - we're logging print statements from a flow
2023-02-23 08:46:32.091 | INFO    | Flow run 'winged-roadrunner' - Created task run 'my_task-0' for task 'my_task'
2023-02-23 08:46:32.093 | INFO    | Flow run 'winged-roadrunner' - Executing 'my_task-0' immediately...
2023-02-23 08:46:32.150 | INFO    | Flow run 'winged-roadrunner' - we're logging print statements from a task
2023-02-23 08:46:32.193 | INFO    | Flow run 'winged-roadrunner' - Finished in state Completed()
2023-02-23 08:46:32.240 | INFO    | Flow run 'winged-roadrunner' - Finished in state Completed('All states completed.')
```
```
prefect flow-run logs 41d847f6-a489-4a9e-b7ba-c27840d81776 --reverse
2023-02-23 08:46:32.240 | INFO    | Flow run 'winged-roadrunner' - Finished in state Completed('All states completed.')
2023-02-23 08:46:32.193 | INFO    | Flow run 'winged-roadrunner' - Finished in state Completed()
2023-02-23 08:46:32.150 | INFO    | Flow run 'winged-roadrunner' - we're logging print statements from a task
2023-02-23 08:46:32.093 | INFO    | Flow run 'winged-roadrunner' - Executing 'my_task-0' immediately...
2023-02-23 08:46:32.091 | INFO    | Flow run 'winged-roadrunner' - Created task run 'my_task-0' for task 'my_task'
2023-02-23 08:46:32.044 | INFO    | Flow run 'winged-roadrunner' - we're logging print statements from a flow
```

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- Closes #8000
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: src/prefect/cli/flow_run.py [hunk 1]
```diff
@@ -196,9 +203,19 @@ async def logs(
...
+                    limit=num_logs_to_return_from_page,
+                    offset=offset,
+                    sort=schemas.sorting.LogSort.TIMESTAMP_DESC,
+                )
+            else:
+                page_logs = await client.read_logs(
+                    log_filter=log_filter,
+                    limit=num_logs_to_return_from_page,
+                    offset=offset,
+                )
```
Comment (madkinsz) [inline src/prefect/cli/flow_run.py:218]: We could probably do this in a single line if statement, e.g.
```suggestion
            page_logs = await client.read_logs(
                log_filter=log_filter,
                limit=num_logs_to_return_from_page,
                offset=offset,
                sort=schemas.sorting.LogSort.TIMESTAMP_DESC if reverse else schemas.sorting.LogSort.TIMESTAMP_ASC
            )
```

**Review by madkinsz — CHANGES_REQUESTED** [review]
This looks like a good start! Thanks for contributing :)

We'll need tests for this at `tests/cli/test_flow_run.py`; there should be some existing flow run log tests there for you to base yours off of.

File: src/prefect/cli/flow_run.py (hunk 1 above)
Comment (l0b64) [reply src/prefect/cli/flow_run.py:218]: np thx for your review

File: src/prefect/cli/flow_run.py [hunk 2]
```diff
@@ -164,6 +165,12 @@ async def logs(
         ),
         min=1,
     ),
+    reverse: bool = typer.Option(
+        None,
```
Comment (madkinsz) [inline src/prefect/cli/flow_run.py:169]: I think this default should be `False`?

Let me know if you need help with the tests!

Comment (l0b64) [reply src/prefect/cli/flow_run.py:169]: well noted I agree default value must be false since reverse is a boolean type

**Comment by l0b64** [comment]
> This looks like a good start! Thanks for contributing :)
> 
> We'll need tests for this at `tests/cli/test_flow_run.py`; there should be some existing flow run log tests there for you to base yours off of.

**Review by billpalombi — APPROVED** [review]
Thank you @lounis89!

@madkinsz, I'm on board with this from a UX perspective.

**Comment by lounis89** [comment]
> Thank you @lounis89!
> 
> @madkinsz, I'm on board with this from a UX perspective.

Thank you both, @madkinsz and @billpalombi, for your valuable feedback. I am grateful for the opportunity to contribute to the Prefect project.

**Review by madkinsz — APPROVED** [review]
Nice!

**Pull request merged**